tools/distribution/verify.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python3
+
+#
```

Comment:
Nit: Python purist might ask for docstring syntax here. I don't feel that strongly (I think I've been using docstring elsewhere FWIW).